packages/ml/src/utils/constants.py

```diff
@@ -0,0 +1,60 @@
+# Define the default values
+import os
+
+from dotenv import load_dotenv
+from langchain.embeddings import HuggingFaceEmbeddings, OpenAIEmbeddings
+from sentence_transformers import SentenceTransformer
+
+
+
+# Set API Keys
+load_dotenv()
+
+ACTIVELOOP_TOKEN = os.getenv("ACTIVELOOP_TOKEN", None)
+os.environ['ACTIVELOOP_TOKEN'] = ACTIVELOOP_TOKEN
+
+OPENAI_API_MODEL = os.getenv("OPENAI_API_MODEL", "gpt-3.5-turbo-0613")
+assert OPENAI_API_MODEL, "OPENAI_API_MODEL environment variable is missing from .env"
+OPENAI_API_KEY = os.getenv("OPENAI_API_KEY", "")
+assert OPENAI_API_KEY, "OPENAI_API_KEY environment variable is missing from .env"
+GOOGLE_CSE_ID = os.getenv("GOOGLE_CSE_ID", "")
+assert GOOGLE_CSE_ID, "GOOGLE_CSE_ID environment variable is missing from .env"
+GOOGLE_API_KEY = os.getenv("GOOGLE_API_KEY", "")
+
+# Set Agent Settings
+AGENT_NAME = os.getenv("AGENT_NAME", "")
+assert AGENT_NAME, "AGENT_NAME variable is missing from .env"
+AGENT_ROLE = os.getenv("AGENT_ROLE", "")
+assert AGENT_ROLE, "AGENT_ROLE variable is missing from .env"
+AGENT_OBJECTIVE = os.getenv("AGENT_OBJECTIVE", "")
+assert AGENT_OBJECTIVE, "AGENT_OBJECTIVE variable is missing from .env"
+
+USE_HF_EMBEDDINGS = False
+USE_LOCAL_STORAGE = True
```

Comment:
If True it means use DeepLake local, otherwise use ActiveLoop, required inform token